src/main/java/com/djrapitops/extension/EssentialsExtension.java

```diff
@@ -25,19 +25,24 @@ of this software and associated documentation files(the "Software"), to deal
 import com.djrapitops.plan.extension.CallEvents;
 import com.djrapitops.plan.extension.DataExtension;
 import com.djrapitops.plan.extension.FormatType;
+import com.djrapitops.plan.extension.NotReadyException;
 import com.djrapitops.plan.extension.annotation.*;
 import com.djrapitops.plan.extension.icon.Color;
 import com.djrapitops.plan.extension.icon.Family;
 import com.earth2me.essentials.Essentials;
 import com.earth2me.essentials.User;
 import org.bukkit.plugin.java.JavaPlugin;
 
+import java.math.BigDecimal;
 import java.util.*;
 
 /**
  * DataExtension for Essentials plugin.
  *
+ * Economy Support by Vankka
+ *
  * @author Rsl1122
+ * @author Vankka
  */
 @PluginInfo(name = "Essentials", iconName = "flask", iconFamily = Family.SOLID, color = Color.DEEP_ORANGE)
 public class EssentialsExtension implements DataExtension {
```

Comment:
Could you add an InvalidateMethod annotation for the removed hasUser so that the old data is removed